templates/install-wp-tests.sh

```diff
@@ -66,18 +96,15 @@ set -ex
 
 install_wp() {
 
-	if [ -d $WP_CORE_DIR ]; then
+	if [ -f $WP_CORE_FILE ]; then
 		return;
 	fi
 
+	rm -rf $WP_CORE_DIR
 	mkdir -p $WP_CORE_DIR
 
 	if [[ $WP_VERSION == 'nightly' || $WP_VERSION == 'trunk' ]]; then
-		mkdir -p $TMPDIR/wordpress-trunk
-		rm -rf $TMPDIR/wordpress-trunk/*
-        check_svn_installed
-		svn export --quiet https://core.svn.wordpress.org/trunk $TMPDIR/wordpress-trunk/wordpress
-		mv $TMPDIR/wordpress-trunk/wordpress/* $WP_CORE_DIR
+		download https://github.com/WordPress/wordpress-develop/archive/refs/heads/master.tar.gz $WP_CORE_DIR
```

Comment:
Seems to be two steps attempted to be done in one here.

The `download` function takes a URL, and a file path ie. it should be something like:
`download https://github.com/WordPress/wordpress-develop/archive/refs/heads/master.tar.gz $TMP_DIR/wordpress.tar.gz`

Once we have the gzipped TAR file locally, then it can be extracted out into the core directory.

See lines 130/131 of this branch to see prior art.

This explains why the [tests](https://github.com/wp-cli/scaffold-command/actions/runs/18306837772/job/52125945664?pr=356) are failing with "is a directory", because it's trying to save a file onto a directory that already exists.